dev/com.ibm.ws.security.audit.source/src/com/ibm/ws/security/audit/encryption/AuditKeyEncryptor.java

```diff
@@ -22,28 +22,21 @@ public class AuditKeyEncryptor {
     private String algorithm = CryptoUtils.MESSAGE_DIGEST_ALGORITHM_SHA;
     private int len = 24;
```

Comment:
we should prob remove this SHA1 reference too by replacing it with line 32-33

Reply:
done!